receiver/filelogreceiver/README.md

```diff
@@ -100,47 +105,23 @@ Many parsers operators can be configured to embed certain followup operations su
 
 ### Time parameters
 
-All time parameters must have the unit of time specified. e.g.: `200ms`, `1s`, `1m`. 
+All time parameters must have the unit of time specified. e.g.: `200ms`, `1s`, `1m`.
 
 ### Log Rotation
 
-File Log Receiver can read files that are being rotated. 
+File Log Receiver can read files that are being rotated.
 
 ## Example - Tailing a simple json file
 
 Receiver Configuration
-```yaml
-receivers:
-  filelog:
-    include: [ /var/log/myservice/*.json ]
-    operators:
-      - type: json_parser
-        timestamp:
-          parse_from: attributes.time
-          layout: '%Y-%m-%d %H:%M:%S'
-```
 
-## Example - Tailing a plaintext file
-
-Receiver Configuration
 ```yaml
 receivers:
   filelog:
-    include: [ /simple.log ]
+    include: [/var/log/myservice/*.json]
     operators:
-      - type: regex_parser
-        regex: '^(?P<time>\d{4}-\d{2}-\d{2} \d{2}:\d{2}:\d{2}) (?P<sev>[A-Z]*) (?P<msg>.*)$'
+      - type: json_parser
         timestamp:
           parse_from: attributes.time
-          layout: '%Y-%m-%d %H:%M:%S'
-        severity:
-          parse_from: attributes.sev
-```
-
-The above configuration will read logs from the "simple.log" file. Some examples of logs that it will read:
-```
-2023-06-19 05:20:50 ERROR This is a test error message
-2023-06-20 12:50:00 DEBUG This is a test debug message
+          layout: "%Y-%m-%d %H:%M:%S"
 ```
```

Comment:
Maybe a merge conflict here? I don't see why we'd change this stuff.

Reply:
It's my format on save it looks like. Is there a formatter at the project level I should have set up.